.circleci/scripts/get-changed-files.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+echo "Last 10 commits current branch:"
+LOG=$(git log --oneline -n 10 "$CIRCLE_BRANCH")
+echo "$LOG"
+
+echo "Last 10 commits develop:"
+LOG=$(git log --oneline -n 10 develop)
+echo "$LOG"
+
+DIFF_RESULT=$(git diff --name-only develop..."$CIRCLE_BRANCH")
+echo "$DIFF_RESULT"
+
+# Store the output of git diff
+git diff --name-only develop..."$CIRCLE_SHA1" >> changed-files/changed-files.txt
```

Comment:
maybe this should be a diff with `origin/develop`? because above there is the code to `git fetch origin develop`, but those commits are not pulled locally

Reply:
thank you for the suggestion, I tried it but it does not seem to make any difference :thinking: 


![Screenshot from 2024-06-04 11-53-08](https://github.com/MetaMask/metamask-extension/assets/54408225/1e50850a-76d5-4a19-929a-4177a1ca31bb)

Reply:
:information_source: the problem was that the branch was way behind the depth. Updating the branch fixed it